redcap/project.py

```diff
@@ -149,6 +157,423 @@ def _call_api(self, payload, typpe, **kwargs):
         rcr = RCRequest(self.url, payload, typpe)
         return rcr.execute(**request_kwargs)
 
+    def __toggle_dataframe(self,format):
+        ret_format = format
+        if format == 'df':
+            try:
```

Comment:
This is actually a good place to put that logic assuming all of the export methods will use it.